### PR TITLE
Create web app for viewing video streams: Project setup

### PR DESCRIPTION
**Pull request by scott97**
This PR contains the setup work for our web app for viewing video streams.

Notes:

- Interactivity is implemented using custom elements created with Lit.
- Vite is used for live reloading during development and for bundling.
- Typescript is used for static type checking
- Formatting and linting is set up in this project (see README)

### CODE REVIEW

File: openfish-webapp/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Openfish Webapp
+Openfish Webapp is a web interface for interacting wit the Openfish API.
```
Comment (trekhopton) [inline openfish-webapp/README.md:2]: *with

File: openfish-webapp/src/vite-env.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (trekhopton) [inline openfish-webapp/src/vite-env.d.ts:1]: Is this really needed?

File: openfish-webapp/src/api.types.ts
Comment (trekhopton) [inline openfish-webapp/src/api.types.ts:1]: I think this file should be separate to this project setup PR. 

You could create another PR "Add API types".

File: openfish-webapp/index.html
Comment (trekhopton) [inline openfish-webapp/index.html:1]: In future I would in create separate PRs for (environment setup), and for (HTML, CSS) etc.

**Review by trekhopton — CHANGES_REQUESTED** [review]

File: openfish-webapp/src/vite-env.d.ts (hunk 2 above)
Comment (scott97) [reply openfish-webapp/src/vite-env.d.ts:1]: It is part of the project starter template, I don't know what it does

File: openfish-webapp/src/api.types.ts
Comment (scott97) [reply openfish-webapp/src/api.types.ts:1]: API types removed

File: openfish-webapp/src/vite-env.d.ts (hunk 2 above)
Comment (scott97) [reply openfish-webapp/src/vite-env.d.ts:1]: Ok, I've found an explanation: https://vitejs.dev/guide/features.html#client-types 
It adds some type definitions for typescript

**Pull request merged**